tests/x11/enlightenment_first_start.pm

```diff
@@ -0,0 +1,36 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "x11test";
+use strict;
+use testapi;
+use utils;
+
+sub run() {
+    my $self = shift;
+    assert_and_click "enlightenment_profile_selection";
+    assert_and_click "enlightenment_assistant_next";
+    assert_and_click "enlightenment_profile_size";
+    assert_and_click "enlightenment_assistant_next";
+    mouse_hide();
+    assert_screen "enlightenment_windowfocus";
+    assert_and_click "enlightenment_assistant_next";
+    mouse_hide();
```

Comment:
I could fix that by doing mouse_hide at the beginning of the test, in order to actually have defined coordinates. Without this, the system has no idea where to move them back in the first assert_click